htmlreport/cppcheck-htmlreport

```diff
@@ -589,7 +610,11 @@ if __name__ == '__main__':
         output_file.write(HTML_HEAD_END.replace("content", "content_index", 1))
 
         output_file.write('\n       <table>')
-        output_file.write('\n       <tr><th>Line</th><th>Id</th><th>CWE</th><th>Severity</th><th>Message</th></tr>')
+        if add_author_information:
+            output_file.write('\n       <tr><th>Line</th><th>Id</th><th>CWE</th><th>Severity</th><th>Message</th><th></th><th>Author</th><th></th><th>Author mail</th><th></th><th>Date (DD/MM/YYYY)</th></tr>')
```

Comment:
this code is not very easy to read. I see empty headers  `<th></th>` I wonder if that is intentional. can it be refactored somehow. there is copy paste with the `else:` code below probably it would be good to avoid that.

Reply:
Yes, I have added empty headers just for the readability of the htmlreport. I can remove them, they have just an aesthetic effect.